requirements.txt

```diff
@@ -1,4 +1,8 @@
 atlalign
```

Comment:
Can we remove this one since as of right now we don't depend on it?

Reply:
Done in [9bc2e8d](https://github.com/BlueBrain/Deep-Atlas/pull/8/commits/9bc2e8d8ca1a6a9c1ef80f94c923e2d142ec3a84).